Fix the documentation on how to rename the server (bsc#1229825) (#3964)
uyuni-project/uyuni-docs: modules/administration/pages/troubleshooting/tshoot-hostname-rename.adoc

== Rename Server

If you need to change the hostname of the {productname} Server, you can do so using the [command]``mgradm server rename`` command.
This command updates the settings in the {postgresql} database and the internal structures of {productname}.

=== Server Configuration

The command takes no mandatory parameter, but can take the new hostname if it is not the one from the container host.

In case any SSL certificate needs to be generated to match the new hostname, the SSL CA password needs to be provided.

> This is safely achieved using a configuration file

Reviewer: ```suggestion
This is safely achieved using a configuration file.
```

> .Procedure: Prepare the configuration file for the SSL CA password

Reviewer: ```suggestion
.Procedure: Preparing the configuration file for the SSL CA password
```

. Write a [path]``config.yaml`` file with content like the following:
+
[source,shell]
----
ssl:
  password: "<THE CA PASSWORD>"
----

.Procedure: Renaming {productname} Server
. Change the network settings of the server on the system level locally and remotely at the DNS server.
    You also need to provide configuration settings for reverse name resolution.
    Changing network settings is done in the same way as with renaming any other system.
. Reboot the {productname} Server to use the new network configuration and to ensure the hostname has changed.

> . On the container host, from the command line, execute the following command. Add [command]``-c config.yaml`` if you created the file to store the SSL CA password:

Reviewer: ```suggestion
. On the container host, from the command line, execute the following command.
  Add [command]``-c config.yaml`` if you created the file to store the SSL CA password:
```

+

[source,shell]
----
mgradm server rename
----

+

If the new hostname is not resolvable, the command fails.

The renaming procedure also takes place during the restart of the server container.

> The logs can be found by running this command:

Reviewer: ```suggestion
The logs can be found by running this command on the container host:
```

== Rename Server

If you need to change the hostname of the {productname} Server, you can do so using the [command]``mgradm server rename`` command.
This command updates the settings in the {postgresql} database and the internal structures of {productname}.

=== Server Configuration

The command takes no mandatory parameter, but can take the new hostname if it is not the one from the container host.

In case any SSL certificate needs to be generated to match the new hostname, the SSL CA password needs to be provided.
This is safely achieved using a configuration file.

.Procedure: Preparing the configuration file for the SSL CA password

> . Write a [path]``config.yaml`` file with content like the following:

Reviewer 2: ```suggestion
. Create [path]``config.yaml`` file with content like the following:

```
